google/ground-android
# [Add feature] Hide "+" button if no layers are present in the active project

Fixes #33 

Steps to verify:

1. Select "example" project
2. "+" button appears as expected
3. Select "project with no layers" project
4. "+" button should no longer be visible









@gino-m   PTAL?
